Northstar.Client/mod/scripts/vscripts/ui/ms_slider.nut

```diff
@@ -0,0 +1,52 @@
+// ModSettings_Slider
```

Comment:
Maybe rename the funcs ( and file ) to start with ns and force @uniboi to use this for the better scrollbars PR

Reply:
For now, there isn't any other use for this, so I'll leave it as MS_Slider to avoid confusion with something else.

If you want, you can change the name to whatever after this gets merged.

Reply:
wdym ScrollbarExt is a very descriptive name and I absolutely know what it's supposed to mean